include/lcfiplus.h

```diff
@@ -162,10 +162,34 @@ class Parameters {
   }
   // copy operator/constructor
   Parameters(const Parameters& ref)
-  : _map(ref._map)
+  : _map()
   , _allowstring(ref._allowstring)
   {
-    *this = ref;
```

Comment:
I think this is calling the assignment operator, which should have done the deep copy as well.
I'll have another look at this.